File: server/routes/api/epics.js
/* So, I'm going to fold and decide that Epics aren't a special
 * case of issue, but their own thing.
 */
const localEpics = require('./local/epics')
const jiraEpics = require('./jira/epics')

module.exports = function (router) {
  router.get('/epics/search', function (req, res) {
    if (req.settings && req.settings.useJira) {
      return jiraEpics.searchEpics(req, res)
    }
    return localEpics.searchEpics(req, res)
  })

  router.get('/epics/:epicId', function (req, res) {
    if (req.settings && req.settings.useJira) {
      return jiraEpics.get(req, res)
    }
    return localEpics.get(req, res)
  })
}

File: server/api/epics.py
from .app import API_APP


# So, I'm going to fold and decide that Epics aren't a special
# case of issue, but their own thing.
@API_APP.route('/epics/search')
def search_epics():
    return ('', 200)


@API_APP.route('/epics/<int:epic_id>')
def get_epic(epic_id):
    return ('', 200)
